src/ui/components/Button/Button.scss

```diff
@@ -1,20 +1,51 @@
 @import '~core/css/inc/mixins';
-@import '~core/css/inc/vars';
+@import '~ui/css/vars';
 
-.Button,
-.Button:link {
-  @include button();
+.Button--neutral {
+  @include button(
+    $background: $neutral-base-color,
+    $background-active: $neutral-active-color,
+    $background-hover: $neutral-hover-color
+  );
 }
 
-.Button--light,
-.Button--light:link {
-  @include button(#fff, $link-color);
+.Button--light {
```

Comment:
I bet there is. I'd rather refactor it in a separate PR, but I think it can be improved.